includes/actions/RevertAction.php

```diff
@@ -113,6 +113,9 @@ public function onSubmit( $data ) {
 	}
 
 	public function onSuccess() {
+		// Wikia change - Add hook for a successful file revert
+		wfRunHooks( 'FileRevertComplete', [ $this->page ] );
```

Comment:
Just curious, do we have any set policy on `wfRunHooks` vs `Hooks::run`, seeing as core MW ditched `wfRunHooks` as a redundant function call?